06-code.Rmd

```diff
@@ -9,6 +9,24 @@ ottrpal::set_knitr_image_path()
 ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g221fb1e0594_0_0")
 ```
 
+## Reproducibility means we don't need to reinvent the wheel!
+
+When you realize something that you need done, you should first use Google and look on GitHub and StackOverflow for is someone else has written something that works really well. Where at all possible, borrow good code and attribute the author -- no need to reinvent the wheel. As we said previously, R has a great community of users who are constantly creating new and great code, often in the form of packages that are ready for you to install and use.
+
+_This is the beauty of reproducibility, if you or someone else makes great code, it can not only be re-run but it can be repurposed!_
+
+### Tips for choosing packages to use:
+
+- Does the package have easy to use documentation to help guide you on how to use it properly? If the package has underdeveloped documentation it may be difficult for you or other to understand and use properly.
```

Comment:
- Does the package have easy to use documentation to help guide you on how to use it properly? If the package has underdeveloped documentation it may be difficult for you or others to understand and use properly.